mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,50 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """
+    All the info that gets validated
+    has been set by the projects
+    bat files.
+    If you are surtain that the info
+    is incorrect, talk to
+    project Supervisor with haste !
+
+    """
+
+    label = "Validate Project Edit Info"
+    optional = True
+    order = pyblish.api.ValidatorOrder
+    hosts = ["maya"]
+
+    def process(self, context):
+        from maya import cmds
+
+        sceneIn = cmds.playbackOptions(query=True, animationStartTime=True)
+        sceneOut = cmds.playbackOptions(query=True, animationEndTime=True)
```

Comment:
Not saying this is wrong, but so you are aware - In Maya, there are two concepts of time. One is the "startTime" and "endTime", the other is is "minTime" and "maxTime".

`startTime` and `endTime` refers to the range slider, whereas `minTime` and `maxTime` refers to the currently playable area with which the animator is working.

We'll need to make sure that what we validate is the same range that is used for actually publishing the animation cache, which may be one or the other.